src/Microsoft.VisualStudio.Editors/OptionPages/GeneralOptionPageResources.resx

```diff
@@ -144,4 +144,13 @@
   <data name="General_NestingSettings_Title" xml:space="preserve">
     <value>File Nesting Settings</value>
   </data>
+  <data name="General_MultiTargetingSettings_Title" xml:space="preserve">
+    <value>Multi-Targeting Settings</value>
+  </data>
+  <data name="General_PreferSingleTargetBuildsOnLaunch" xml:space="preserve">
+    <value>Prefer single-target builds when starting a project.</value>
+  </data>
+  <data name="General_PreferSingleTargetBuildsOnLaunchDescription" xml:space="preserve">
+    <value>When starting a project that targets multiple .NET frameworks the build time can be reduced by limiting the build to a single framework.</value>
```

Comment:
My English sense might be wrong here, but should this have a comma? I think the rule of thumb is that there should be a comma if you can take the first part of the sentence and move it to the end. Ah, found it. It is called an [introductory clause (2nd one)](https://www.iue.edu/student-success/coursework/commas.html):
```suggestion
    <value>When starting a project that targets multiple .NET frameworks, the build time can be reduced by limiting the build to a single framework.</value>
```